### PR TITLE
CVE-2015-6779 and CVE-2016-1689

### PR DESCRIPTION
**Pull request by JoshVerDow**
Contributing research information on two Chromium vulnerabilities.

### CODE REVIEW

File: cves/CVE-2015-6779.yml [hunk 1]
```diff
@@ -11,7 +11,7 @@ curated_instructions: |
   integrity checks on this file to make sure you fill everything out properly.
   If you are a student, we cannot accept your work as finished unless curated is
   set to true.
-curated: false
+curated: true
 announced_instructions: |
   Was there a date that this vulnerability was announced to the world? You can
   find this in changelogs, blogs, bug reports, or perhaps the CVE date. A good
```
Comment (CatherineOsadciw) [inline cves/CVE-2015-6779.yml]: Remove timestamp from "announced" below (leave date)

File: cves/CVE-2015-6779.yml [hunk 2]
```diff
@@ -35,14 +35,14 @@ description_instructions: |
 description: |
   The issue with this CVE entry is that when a user opens a pdf file in the
```
Comment (CatherineOsadciw) [inline cves/CVE-2015-6779.yml]: Delete "This issue with this CVE entry is that"

File: cves/CVE-2015-6779.yml [hunk 3]
```diff
@@ -151,9 +150,9 @@ major_events:
 
     The event doesn't need to be directly related to this vulnerability, rather,
     we want to capture what the development team was dealing with at the time.
-  answer: 'I did not see any major events, this was more or less an unfortunate bug.
-
-'
+  answer: I did not see any major events, the history of this vulernability was
```
Comment (CatherineOsadciw) [inline cves/CVE-2015-6779.yml]: Spelling

File: cves/CVE-2016-1689.yml [hunk 4]
```diff
@@ -11,7 +11,7 @@ curated_instructions: |
   integrity checks on this file to make sure you fill everything out properly.
   If you are a student, we cannot accept your work as finished unless curated is
   set to true.
-curated: false
+curated: true
 announced_instructions: |
   Was there a date that this vulnerability was announced to the world? You can
   find this in changelogs, blogs, bug reports, or perhaps the CVE date. A good
```
Comment (CatherineOsadciw) [inline cves/CVE-2016-1689.yml]: Remove timestamp from "announced" below (leave date)

File: cves/CVE-2016-1689.yml [hunk 5]
```diff
@@ -226,6 +230,8 @@ mistakes:
   answer: |
     This vulnerability seemed to stem from a lack of design around the desired feature. This
     lack of design then caused the software to be implemented to trust inputs which shouldn't
-    have been trusted. It also seemed that this software was created for temporary use, which
-    is questionable. It doesn't seem to make sense to create software which you know later you
-    are going to redo, why not just design, and create the feature completely the first time?
+    have been trusted. It also seemed that this software was created for temporary use, due
+    to the fact that the team was waiting for a specific library (libyuv::ARGBToA) to become 
+    available for use.  This is understandable to an extent, as companies cannot always wait
```
Comment (olucomedy) [inline cves/CVE-2016-1689.yml]: This is a nice explanation of the the mistakes that were made.

File: cves/CVE-2015-6779.yml [hunk 6]
```diff
@@ -228,14 +228,9 @@ mistakes:
...
-    have to say maintainability. But I do not want to even call it that. Here
-    is why.
-
-    This mistake should not have happened. It is prohibited in HTML, and it
-    isn't in the nature of PDF documents to allow this behavior. Somewhere
-    along the way, I am guessing some code got implemented that made this
-    happen. So I will call it maintainability. but at the end of the day,
-    nobody would think to plan or block for this. It was just all around
-    unlucky and unfortunate that this happened.
+    I would say that this vulnerability was a culmination of design mistakes and
```
Comment (olucomedy) [inline cves/CVE-2015-6779.yml]: Could you give an example of a design mistake that was made?

**Comment by allisonsuarez** [comment]
2 up votes 👍

File: cves/CVE-2015-6779.yml (hunk 6 above)
Comment (JoshVerDow) [reply cves/CVE-2015-6779.yml]: I assume a design mistake was made, due to the default nature of HTML and PDF not allowing this behavior.  For this behavior to of been possible, I am guessing that a design decision was made to allow it.  Though, I did not find any specific example that can verify this.

**Pull request merged**